src/lib/radio/radio.ts

```diff
@@ -280,10 +300,26 @@ export class MdRadioButton implements OnInit {
   name: string;
 
   /** Used to set the 'aria-label' attribute on the underlying input element. */
-  @Input('aria-label') ariaLabel: string;
+  _ariaLabel: string;
+  @Input('aria-label')
+  get ariaLabel(): string {
+    return this._ariaLabel;
+  }
+  set ariaLabel(value: string) {
+    this._ariaLabel = value;
+    this._change.markForCheck();
```

Comment:
I didn't think calling `markForCheck` for every property shouldn't be necessary; for anything that is an `@Input` Angular will do change detection as needed when the value is changed via a template binding. The only thing I know of that needs to change is that the `writeValue` function needs to call `markForCheck`, since the template binding is on `ngModel` (which isn't an `@Input` of the component).